lib/ui/window/pointer_data.h

```diff
@@ -9,6 +9,21 @@
 
 namespace flutter {
 
+// Must match the button constants in events.dart.
+enum PointerButton : int64_t {
```

Comment:
Why is this one enum instead of an enum for each pointer type?

Reply:
Fixed